target/linux/ramips/image/mt7621.mk

```diff
@@ -142,6 +142,16 @@ define Device/newifi-d1
 endef
 TARGET_DEVICES += newifi-d1
 
+define Device/newifi-d2
+  DTS := Newifi-D2
+  IMAGE_SIZE := $(ralink_default_fw_size_16M)
+  DEVICE_TITLE := Newifi D2
+  DEVICE_PACKAGES := \
+	kmod-mt7603 kmod-mt76x2 kmod-sdhci-mt7620 \
+	kmod-usb3 kmod-usb-ledtrig-usbport wpad-mini
```

Comment:
drop wpad-mini. it is included by default.

drop kmod-usb-ledtrig-usbport. I neither see an usb led in the DTS nor a set_usb_led() call in 01_leds